hoomd/jit/CMakeLists.txt

```diff
@@ -42,6 +42,10 @@ pybind11_add_module(_${PACKAGE_NAME} SHARED ${_${PACKAGE_NAME}_sources} ${_${PAC
 add_library(HOOMD::_${PACKAGE_NAME} ALIAS _${PACKAGE_NAME})
 
 add_library (_${PACKAGE_NAME}_llvm SHARED ${_${PACKAGE_NAME}_llvm_sources})
+if (CMAKE_CXX_STANDARD LESS 14)
+    set_property(TARGET _${PACKAGE_NAME}_llvm PROPERTY CXX_STANDARD 14)
+endif()
+set_property(TARGET _${PACKAGE_NAME} PROPERTY CXX_STANDARD 14)
```

Comment:
Why is this outside the `if`?

Reply:
Sorry about this, just fixed it. Not sure why the AMD GPU tests are failing.